test/micro/org/openjdk/bench/java/lang/reflect/Clazz.java

```diff
@@ -52,6 +55,34 @@ public Constructor getConstructor() throws NoSuchMethodException {
         return Clazz.class.getConstructor();
     }
 
+    /**
+     * Get modifiers for this class through reflection
+     *
+     * @return
+     * @throws NoSuchMethodException
+     */
+    @Benchmark
+    public int getModifiers() throws NoSuchMethodException {
```

Comment:
The only `Throwable`s that can be thrown by calling `Class::getModifiers()` are `Error`s (e.g.: `StackOverflowError`) and `RuntimeException`s (e.g.: `NullPointerException`):
```suggestion
    public int getModifiers() {
```